modules/post-installation-adding-nutanix-failure-domains-compute-machines.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="post-installation-adding-nutanix-failure-domains-compute-machines_{context}"]
+= Distributing compute machines across failure domains
+
+You can distribute compute machines across Nutanix failure domains by:
+
+* Modifying existing compute machine sets.
+* Creating new compute machine sets.
+
+The following procedure details how to distribute compute machines across failure domains by modifying existing compute machine sets. For more information on creating a compute machine set, see "Additional resources".
+
+.Prerequisites
+
+* You have configured the failure domains in the cluster's Infrastructure custom resource (CR).
+
+.Procedure
+
+. Run the following command to view the cluster's Infrastructure CR.
++
+[source,terminal]
+----
+$ oc describe infrastructures.config.openshift.io cluster
+----
+. For each failure domain (`platformSpec.nutanix.failureDomains`), note the cluster's UUID, name, and subnet object UUID. These values are required to add a failure domain to a compute machine set.
+. List the compute machine sets in your cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get machinesets -n openshift-machine-api
+----
+. Edit the first compute machine set by running the following command:
++
+[source,terminal]
+----
+$ oc edit machineset <machineset_name> -n openshift-machine-api
+----
+. Configure the compute machine set to use the the first failure domain by adding the following to the `spec.template.spec.providerSpec.value` stanza:
```

Comment:
Same here, maybe?
```suggestion
. Configure the compute machine set to use a first failure domain by adding the following to the `spec.template.spec.providerSpec.value` stanza:
```

Reply:
This was raised early as part of an SME review. The decision was to go with CR.

https://github.com/openshift/openshift-docs/pull/69618#discussion_r1457598396

Reply:
I removed the extra `the`.